src/Microsoft.VisualStudio.Editors/MyApplication/MyApplicationProperties.vb

```diff
@@ -593,6 +596,34 @@ Namespace Microsoft.VisualStudio.Editors.MyApplication
             End Set
         End Property
 
+        Friend Property HighDpiMode As Integer Implements IMyApplicationPropertiesInternal.HighDpiMode
+            Get
+                Return _myAppData.HighDpiMode
+            End Get
+            Set
+                Select Case Value
+                    Case _
+                    0, ' There is no ApplicationService.HighDpiMode available to do it more clean.
+                    1,
+                    2,
+                    3,
+                    4
+                        ' Valid - continue
+                    Case Else
+                        Throw New ArgumentOutOfRangeException(NameOf(Value))
+                End Select
```

Comment:
I am not sure what the/your intention here is.
So, if you - and I am just assuming! 😄 - just wanted to test of correct range, you can just test like this:

```suggestion
               If Value < 0 Or Value > 4
                        Throw New ArgumentOutOfRangeException(NameOf(Value))
                End If
```

You could also use `To` as a range operator in VB:

```suggestion
                Select Case Value
                    Case 0 to 4 ' Valid: continue
                    Case Else
                        Throw New ArgumentOutOfRangeException(NameOf(Value))
                End Select
```